certbot/docs/using.rst

```diff
@@ -707,6 +707,11 @@ form is not appropriate to run daily because each certificate will be
 renewed every day, which will quickly run into the certificate authority
 rate limit.)
 
+Certbot provides the environment variables `RENEWED_DOMAINS` and `FAILED_DOMAINS`
```

Comment:
We can wait right before merging to change this, but I think we should mention the version number where this feature became available. There's a lot of old Certbots entombed in various distros out there.

Reply:
I ran off the checklist and assumed I'd need to update all the docs. I can revert if you wish?

Reply:
This is a response to https://github.com/certbot/certbot/pull/9724/files#r1243102591 just in case GitHub's UI doesn't make that clear.

Sorry! I wasn't clear in my previous comment. I was trying to suggest that we add a phrase to the beginning of what you wrote here like "Starting with Certbot 2.7.0, ...". My thinking is that there are plenty of old versions of Certbot out there that people are using which will not have this behavior. 

I also said that we can wait until right before we merge this to make this change if you want to avoid having to update it again if this PR isn't included in our next release.

Reply:
I'll take the risk that this is merged for 2.7.0 and update accordingly.